addOns/automation/src/main/javahelp/org/zaproxy/addon/automation/resources/help/contents/job-exitstatus.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<TITLE>
+Automation Framework - exitStatus Job
+</TITLE>
+</HEAD>
+<BODY>
+<H1>Automation Framework - exitStatus Job</H1>
+
+This job sets the exit code based on scan results.
+It also allows you to choose which exit values are used.
+<p>
+If warnLevel or errorLevel are set then the job will report a warning or error if any alerts
+are raised which have the same risk level or greater.
+<p>
+By default when ZAP is run with the <code>-cmd</code> and <code>-autorun</code> options then it will
+exit with a 1 if there are any errors, with a 2 if there are any warnings, and if everything is ok
```

Comment:
As we've seen elsewhere this week and in the user group. We need to be clear that this is jobs that result in errors or warnings. Not log messages, or any other interpretation of Alerts than specifically defined.

Reply:
Technically is not just jobs but anything related to the plan, it might be alert/stats tests, the context/env, scripts, and so on.